src/components/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import { ReactComponent as FooterLogo } from '../assets/sign.svg';
+
+const FooterWrapper = styled.footer`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  height: 100px;
+  width: 100%;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  width: 100%;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 250px;
```

Comment:
Responsiveness is not required in the acceptance criteria, but it's always good to keep it in mind. Setting a fixed padding especially with this size makes the app incompatible with mobile devices. See the screenshot below. It's better to use a `max-width` together with the `width: 100%` and `margin: auto`. The max-width of the footer should be `1440px - 2 * 250px = 940px`. So using a `max-width: 980px` plus a small `padding: 0 20px` will make it look nice on mobile as well.